packages/bot-web-ui/src/components/dashboard/quick-strategy/quick-strategy-components/container.tsx

```diff
@@ -37,7 +37,11 @@ const QuickStrategyContainer = (props: TQuickStrategyProps) => {
                     component: <MarketOption key={symbol.text} symbol={symbol} />,
                     ...symbol,
                 }))
-                .filter(option => option.group !== 'Cryptocurrencies'), // Until Crypto enabled for Dbot
+                // Until Crypto enabled for Dbot
+                .filter(option => option.group !== 'Cryptocurrencies')
+                .filter(
+                    option => option.text !== 'Volatility 150 (1s) Index' && option.text !== 'Volatility 250 (1s) Index'
```

Comment:
Instead of filtering it directly here. Can we have an array of filtered items and loop over here? Also can we not filter it by `text` there's still a small chance that these titles might change and will affect this filter.  Can we go with the `symbol` instead?